added a contributing.md

Kanchan253 opened the pull request:
Hi, I have added a contributing.md file which has the steps on how to contribute to the project. Please check and review it and let me know if I have to make any changes in it. Thank you!

cskanchandaryanani committed:
added a contributing.md

silenteyesoncode commented on Contributing.md:
```diff
@@ -0,0 +1,70 @@
+●Go to your GitHub account and open the forked repository.
+●Click the "Code" button and then the copy icon to copy the repository's URL.
+
+![Alt text](../../Users/Lenovo/Documents/code.png)
```
It appears that there is an issue with the image URL provided in the code snippet. The path `"../../Users/Lenovo/Documents/fork.png"` suggests a local file path on your personal computer, which cannot be accessed or displayed on the internet.

Kanchan253 replied:
Should I add the images to a file in the project?

silenteyesoncode replied:
yes you can upload the image in GitHub itself here

silenteyesoncode commented on Contributing.md:
```diff
@@ -0,0 +1,70 @@
+
+●Open the terminal and change your directory to the cloned repository.
+
+cd AI-Powered-News-Article-Classifier-with-History/
```
Can we have to different path command for backend and frontend folders

Kanchan253 replied:
So I have to add a separate path for the backend folder for the people who want to contribute to the backend and the same for the frontend part of it?

silenteyesoncode replied:
Yes, U got me

silenteyesoncode reviewed (changes requested):
Hi @Kanchan253 

I Appreciate the efforts here, although with respect to open source repository standers i may need some changes here 
you can take reference from  [OpenMind](https://github.com/The-Unleashed-Club/openMind/blob/main/Contributing.md) repo 

###  [Feel free to react out to me for any query ](https://calendly.com/7silenteyes07/professional-coffee-chat)

cskanchandaryanani committed:
Update:Contributing.md

silenteyesoncode commented on Contributing.md:
```diff
@@ -0,0 +1,93 @@
+
+●Click the "Fork" button at the top right corner of this repository's page.
+
+![Alt text](images/fork.png)
```
can you please remove Alt text
and add `<img src="YOUR_IMAGE_URL"  width="{Between_300 - 500_px}" >`

silenteyesoncode replied:
If you give me the edit access I can show you
@Kanchan253

Kanchan253 replied:
@silenteyesoncode I have given you access.

silenteyesoncode replied:
great i'll made some changes and Merge the PR By Monday

Kanchan253 replied:
Ok sure

silenteyesoncode replied:
Thanks you so much for your contribution. I'll show your profile in in repo readme for sure 
Wish you a good luck

Kanchan253 replied:
Thank you!!

silenteyesoncode committed:
Update Contributing.md

silenteyesoncode committed:
Update Contributing.md

using Image Tag

silenteyesoncode committed:
Update Contributing.md

@Kanchan253  I notice you have a very little knowledge regarding MD writing 
try to read my changes as a raw code

silenteyesoncode committed:
Merge branch 'silenteyesoncode:main' into contributing-steps
